### PR TITLE
feat: add tooltip to Scheduled badge

### PR DESCRIPTION
**Pull request by andreas-unleash**
Closes # [1-1831](https://linear.app/unleash/issue/1-1831/enhancement-tooltips-for-change-request-overview-page-to-add-hover)

Screenshot 2023-12-22 at 12 59 15
Screenshot 2023-12-22 at 12 59 07

### CODE REVIEW

File: frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
     Close,
     Info,
 } from '@mui/icons-material';
+import { HtmlTooltip } from '../../common/HtmlTooltip/HtmlTooltip';
```
Comment (kwasniew) [inline frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx:11]: components/common

File: frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx [hunk 2]
```diff
@@ -66,10 +67,21 @@ export const ChangeRequestStatusBadge: VFC<IChangeRequestStatusBadgeProps> = ({
                 ) : (
                     <Info fontSize={'small'} />
                 );
+            const scheduledAt = new Date(
+                schedule?.scheduledAt || '',
```
Comment (kwasniew) [inline frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx:71]: new Date("") gives invalid date

Comment (kwasniew) [reply frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx:71]: why is scheduledAt optional in schedule?

File: frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx [hunk 3]
```diff
@@ -66,10 +67,21 @@ export const ChangeRequestStatusBadge: VFC<IChangeRequestStatusBadgeProps> = ({
...
+            ).toLocaleString();
+
+            const tooltipTitle =
+                schedule?.status === 'pending'
```
Comment (kwasniew) [inline frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx:75]: why is schedule optional here?

Comment (andreas-unleash) [reply frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx:75]: It's not really optional when the state is Scheduled. Just a result of the type - will improve it with a union

**Review by kwasniew — APPROVED** [review]

**Pull request merged**